app/config/__init__.py

```diff
@@ -6,15 +6,15 @@
 
 
 class Config(object):
-    CONTACT_EMAIL = os.environ.get("CONTACT_EMAIL")
-    CONTACT_PHONE = os.environ.get("CONTACT_PHONE")
-    DEPARTMENT_NAME = os.environ.get("DEPARTMENT_NAME", "LAA")
-    DEPARTMENT_URL = os.environ.get("DEPARTMENT_URL")
+    CONTACT_EMAIL = os.environ.get("CONTACT_EMAIL", "")
+    CONTACT_PHONE = os.environ.get("CONTACT_PHONE", "")
+    DEPARTMENT_NAME = os.environ.get("DEPARTMENT_NAME", "MOJ Digital")
+    DEPARTMENT_URL = os.environ.get("DEPARTMENT_URL", "https://mojdigital.blog.gov.uk/")
     RATELIMIT_HEADERS_ENABLED = True
     RATELIMIT_STORAGE_URI = os.environ.get("REDIS_URL")
-    SECRET_KEY = os.environ.get("SECRET_KEY")
+    SECRET_KEY = os.environ.get("SECRET_KEY", "CHANGE_ME")
```

Comment:
How does this work, just worried is there a chance that defaulting this and not entering a secret key environment would make us vulnerable? Would it throw an error if we didn't default it or still just continue?

Reply:
Yes, I think we should definitely change this to raise an exception if required settings are not defined. You can do this by doing os.environ["KEY"] and if the "KEY" does not exist it will raise an exception

Reply:
Updated to reflect this